packages/backend/src/open_payments/receiver/service.ts

```diff
@@ -296,10 +296,20 @@ async function getIncomingPaymentGrant(
   const existingGrant = await deps.grantService.get(grantOptions)
```

Comment:
L241 in this service we could update from
`accessToken: grant.accessToken || ''` -> `accessToken: grant.accessToken`